components/org.wso2.carbon.metrics.impl/src/main/java/org/wso2/carbon/metrics/impl/metric/collection/MeterCollection.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
@prabathabey It was already been fixed in [commit:0402c09](https://github.com/wso2/carbon-metrics/commit/d8ad48ed53fd2c90ebcfe5923a3ae3e00215bc43).